docs/release/trg-0/trg-7-2.md

```diff
@@ -0,0 +1,90 @@
+---
+title: TRG 7.02 - License and Copyright header
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: "mandatory after": 19th of May 2023
```

Comment:
Those changes are so easily done and should already be followed, i wouldn't add a mandatory after (probably all other TRG 7ts too)

Reply:
Removed it for TRG 7.01-7.04, 5+6 are new.